sorc/ocean_merge.fd/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+list(APPEND fortran_src
+  merge_lake_ocnmsk.f90
+)
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -r8 -i4 -convert big_endian")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fdefault-real-8")
+  if(CMAKE_Fortran_COMPILER_VERSION VERSION_GREATER_EQUAL 10)
+    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fallow-argument-mismatch")
+  endif()
+endif()
+
+set(exe_name ocean_merge)
+add_executable(${exe_name} ${fortran_src})
+target_link_libraries(
+  ${exe_name}
+  sp::sp_d
+  NetCDF::NetCDF_Fortran)
```

Comment:
The NCEPLIBS group is trying to get rid of the sp library. I don't see where sp is used by this code.

Reply:
I did not check for redundant libraries.  I just removed that and was able to compile successfully.